runners/flink/1.10/build.gradle

```diff
@@ -17,6 +17,10 @@
  */
 
 def basePath = '..'
+def flink_clients_version = "1.10.1"
+def flink_annotations_version = "1.10.1"
+def flink_optimizer_version = "1.10.1"
```

Comment:
These flink versions are set up just below here in the `project.ext` section.

@mxm may have some comment or recommend someone for a comment.

Reply:
Done as per your recommendation :)